themes/pioneer/layouts/partials/analytics.html

```diff
@@ -0,0 +1,11 @@
+{{ if and .Site.Config.Services.GoogleAnalytics.ID (not hugo.IsServer) }}
+{{ template "_internal/google_analytics.html" . }}
+{{/* Add support for umami website analytics */}}
+{{ else if and .Site.Params.umami.enabled (not hugo.IsServer) }}
```

Comment:
OK, the error is in these conditional statements. It works if I remove them. 
Not that strange since those 4 lines are copy-pasted and I don't gork them.